src/Illuminate/Foundation/Vite.php

```diff
@@ -233,7 +233,7 @@ public function useStyleTagAttributes($attributes)
     /**
      * Use the given callback to resolve attributes for preload tags.
      *
-     * @param  (callable(string, string, ?array, ?array): array|false)|array|false  $attributes
+     * @param  (callable(string, string, ?array, ?array): (array|false))|array|false  $attributes
```

Comment:
This change was required for PHPStan parser to correctly understand the return types of the callable.